examples/chip-tool/commands/common/Command.cpp

```diff
@@ -32,6 +32,10 @@
 #include <lib/support/ScopedBuffer.h>
 #include <lib/support/logging/CHIPLogging.h>
 
+#define NO_OPTIONAL_ARGS 0
+#define OPTIONAL_ARGS_OFFSET 2
```

Comment:
Please document that this is because each optional arg is two entries in argv: name and value.